src/components/LoginComponent.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import "../styles/form.style.scss"
+import {Formik} from "formik"
+import * as yup from "yup"
+import { useNavigate } from 'react-router-dom'
+
+const mySchema = yup.object().shape({
+  Email: yup.string().required("Username is requires!"),
+  Password: yup.string().min(8, "login must be at least 8 chars long!")
+})
+
+function LoginComponent() {
+  const navigate = useNavigate()
+
+  function submitForm(values,action){
+    console.log(values)
+    navigate("/dashboard")
+  }
+
+  return (
+    <Formik 
+      initialValues={{
+      Email: "",
+      Password: "",
+      }}
+      onSubmit = {submitForm}
+      validationSchema = {mySchema}
+
+    >
+      {
+        ({values,errors,handleSubmit,handleChange})=> {
+          return (
+            <div className='container'>
+  
+              <form autoComplete='off' onSubmit={handleSubmit}>
+                <label htmlFor = "email">Username</label>
+                <input type = "email" name = "Email" id ='email' onChange={handleChange} value = {values.Email}/>
+                <p>{errors.Email}</p>
```

Comment:
Create separate component for errors, should be used like this:
<FormError error={errors.email} />